Poker.BE/Poker.BE.Domain/Game/Turn.cs

```diff
@@ -12,58 +12,34 @@ public class Turn
     {
         #region Fields
         private Player currentPlayer;
-        private Pot currentPot;
```

Comment:
no. it's just useless for the turn to have it...
it has one job for now, the onTurnChanged event